src/Microsoft.Bcl.HashCode/src/HashCode.cs

```diff
@@ -0,0 +1,423 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
I only brought in the history from the `src/Microsoft.Bcl.HashCode/` directory in corefx. This file was not part of that folder so I had to add it manually. It was originally located in https://github.com/dotnet/corefx/blob/b86ce69c316b4b088356b89bab80f7fa79ff799c/src/Common/src/CoreLib/System/HashCode.cs